packages/jwt-status-list/src/status-list.ts

```diff
@@ -31,6 +31,14 @@ export class StatusList {
     this.totalStatuses = statusList.length;
   }
 
+  /**
+   * Get the status list.
+   * @returns
```

Comment:
This misses the type. Or it should be removed.

Reply:
returns got removed, also for the other functions. Since each function has a typed return, it should be enough.